contracts/hts-precompile/IHRC.sol

```diff
@@ -1,7 +1,15 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.8.7;
+pragma solidity >=0.4.9 <0.9.0;
```

Comment:
Can't make changes to this file as it's purpose is to serve as a representation of the system contract redirect calls.
Additions would be the result of an accepted HIP modifying the services

Reply:
I see, but why was it ever locked specifically to solidity 0.8.7? interfaces are generally permissive and the other related interfaces e.g. `IHederaTokenService` do specify `>=0.4.9 <0.9.0;`

Reply:
@Nana-EC is your issue with the updated solidity version? or with the added interfaces to this file? The IHRC interface is compatible will all versions in the updated range so that shouldn't be an issue, but I can move the added interfaces into a separate file.

Reply:
@mshakeg Ah no issue with the solidity version. 
We can modify that for compatibility.
Issue is with adding more code to it that doesn't reflect what the consensus nodes actually support.